test/integration/basic_dataplane_standalone_test.go

```diff
@@ -139,7 +139,7 @@ func createFixture(test *testing.T, timeout time.Duration) *standaloneDataplaneF
 		test:    test,
 	}
 
-	k8s, err := kube_testing.NewK8s(true)
+	k8s, err := kube_testing.NewK8s(true, false)
```

Comment:
Let's by default cleanup all, why we need to pass it

Reply:
OK, removed the corresponding commit.